package-lock.json

```diff
@@ -24087,9 +24087,9 @@
       }
     },
     "ansi-colors": {
-      "version": "4.1.1",
-      "resolved": "https://registry.npmjs.org/ansi-colors/-/ansi-colors-4.1.1.tgz",
-      "integrity": "sha512-JoX0apGbHaUJBNl6yF+p6JAFYZ666/hhCGKN5t9QFjbJQKUU/g8MNbFDbvfrgKXvI1QpZplPOnwIo99lX/AAmA=="
+      "version": "4.1.3",
+      "resolved": "https://registry.npmjs.org/ansi-colors/-/ansi-colors-4.1.3.tgz",
+      "integrity": "sha512-/6w/C21Pm1A7aZitlI5Ni/2J6FFQN8i1Cvz3kHABAAbw93v/NlvKdVOqz7CCWz/3iv/JplRSEEZ83XION15ovw=="
```

Comment:
@voxpelli from https://github.com/mochajs/mocha/pull/5150#pullrequestreview-2091274231:

> The one change looks indeed harmless: [doowb/ansi-colors@4.1.1...4.1.3](https://github.com/doowb/ansi-colors/compare/4.1.1...4.1.3)
> 
> But can you do a new `npm install ansi-colors@^4.1.3` to ensure that our lockfile actually also references it?

This brings up an interesting point: we can either:
* include the newer `4.1.3`, and miss out on testing `4.1.1`
* keep the older `4.1.1`, and miss out on testing `4.1.3`

🤷 I don't mind either way, and do like keeping to the latest internally. Just thought I'd note it. I think we're on the same page 😄

Reply:
If we do `^4.1.3` we get an update as well, then we need a lockfile maintenance job to ensure we continuously test against what our users are getting: https://docs.renovatebot.com/configuration-options/#lockfilemaintenance

Reply:
Updated to the latest. 👍